client/lib/purchases/index.ts

```diff
@@ -422,6 +422,16 @@ export function isCancelable( purchase: Purchase ) {
 	return purchase.canDisableAutoRenew;
 }
 
+/**
+ * Checks if a purchase is attached to a siteless holding site
+ *
+ * @param purchase The purchase to check
+ *
+ * @returns {boolean} True if the purchase is attached to a siteless holding site
+ */
+export function isSiteless( purchase: Purchase ): boolean {
+	return purchase.domain.includes( 'siteless' );
```

Comment:
I wonder if this is too general? What would happen if I purchased a domain registration for the domain `examplesiteless.com`?

Reply:
Hmmmm, that's a good point, I'll see if there's better data available to determine if it is a siteless site. I could just check specifically for `siteless.jetpack.com` or `siteless.akismet.com` if there isn't

Reply:
I found some existing functions that handle this, thank you for pointing this out